contrib/modelci/README.md

```diff
@@ -0,0 +1,149 @@
+# Model CI
+
+The users occasionally found a negligible performance or precision issue between different Paddle versions. Though we have unit tests for each class and Travis-CI to ensures the precision of each operator, there is no any logic to ensure the model (a composition of several operators) works as reliable as the operators.
+
+There are several conditions where an existing model will fail either in performance or precision:
+
+1. the incomplete coverage test cases, such as lacking the test of precision. 
```

Comment:
This should be `Incomplete coverage of test cases. For example, tests lacking precision check.`